packages/components/src/wordpress-watermark/index.tsx

```diff
@@ -0,0 +1,26 @@
+interface WatermarkProps {
+	className?: string;
+	size?: { width: number; height: number };
+}
+
+const WordPressWordmark: React.FunctionComponent< WatermarkProps > = ( {
+	className = 'wpcom-wordmark',
+	size = { width: 150, height: 25 },
+} ) => {
+	return (
+		<svg
+			className={ className }
+			width={ size.width }
+			height={ size?.height }
+			viewBox="0 0 1007.7 172.4"
+			xmlns="http://www.w3.org/2000/svg"
+		>
+			<g fill="#FFF">
```

Comment:
This should be configurable and default to `#fff`. Currently, we're override this value [with CSS](https://github.com/Automattic/wp-calypso/blob/trunk/client/layout/universal-navbar-header/nav-style.scss#L99-L101). 

You'd only need to pass it [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/layout/universal-navbar-header/index.jsx#L47) and the rest can default to white.